src/RadixDlt.NetworkGateway.DataAggregator/Configuration/LedgerConfirmationOptions.cs

```diff
@@ -71,35 +71,12 @@ namespace RadixDlt.NetworkGateway.DataAggregator.Configuration;
 
 public sealed record LedgerConfirmationOptions
 {
-    /// <summary>
-    /// Gets or sets CommitRequiresNodeQuorumTrustProportion.
-    /// Requires at least this proportion of enabled nodes to commit (by trust weighting).
-    /// </summary>
-    [ConfigurationKeyName("CommitRequiresNodeQuorumTrustProportion")]
-    public decimal CommitRequiresNodeQuorumTrustProportion { get; set; } = 0.51m;
-
-    /// <summary>
-    /// Gets or sets OnlyUseSufficientlySyncedUpNodesForQuorumCalculation.
-    /// If enabled, the quorum calculation only takes account of nodes which are considered "sufficiently synced up".
-    /// </summary>
-    /// <seealso cref="SufficientlySyncedStateVersionThreshold"/>
-    [ConfigurationKeyName("OnlyUseSufficientlySyncedUpNodesForQuorumCalculation")]
-    public bool OnlyUseSufficientlySyncedUpNodesForQuorumCalculation { get; set; } = true;
-
-    /// <summary>
-    /// Gets or sets SufficientlySyncedThreshold.
-    /// A node is considered sufficiently synced up if its reported top of ledger state version is with this
-    /// many transactions of the top of the db ledger (or ahead of the db ledger).
-    /// </summary>
-    [ConfigurationKeyName("SufficientlySyncedStateVersionThreshold")]
-    public long SufficientlySyncedStateVersionThreshold { get; set; } = 1000;
-
     /// <summary>
     /// Gets or sets MaxCommitBatchSize.
     /// The maximum batch to send to the ledger extension service for committing.
     /// </summary>
     [ConfigurationKeyName("MaxCommitBatchSize")]
-    public long MaxCommitBatchSize { get; set; } = 300;
+    public long MaxCommitBatchSize { get; set; } = 1000;
```

Comment:
for some reason we had different value in json file, i see no point in having different value in these 2 files.

Reply:
It's a breaking change. Please revert that.

Reply:
how is it breaking? it was set to 1000 in .json file and was effective for all environments. https://github.com/radixdlt/babylon-gateway/pull/756/files#diff-3ae0868be67e7c63004cb1768b3818f53803743a181be9833ed1322d5a0acaf6L38